### PR TITLE
Add iOS add-to-app docs to sidenav

### PR DESCRIPTION
**Pull request by xster**
Split the iOS staging docs and add them to the sidenav

### CODE REVIEW

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+---
+
+TODO(jmagman): add the pre-built framework alternative and review.
+
+## Integration steps
+
+To integrate a Flutter module into your app, use the following steps:
+
+ 1. [Create Flutter Module](#create-a-flutter-module)
+    Creates the Flutter project which is hosted in the existing app.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: which => that

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+ 1. [Add Module Dependency in Existing
+    App](#make-the-host-app-depend-on-the-flutter-module)
+    Sets up XCode to include your Flutter project into the build of your
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: I realize this error was already here, but it's "Xcode", not "XCode". (Global change)

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Flutter projects created using `flutter create xxx` include simple host apps for
+your Flutter/Dart code (a single-ViewController iOS host). You can modify these
+host apps to suit your needs and build from there.
+
+If you're starting off with an *existing* host app for either platform, you'll
+likely want to include your Flutter project in that app as some form of library
+instead.
+
+In order to embed Flutter like a library, use the Flutter module template.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: "like a library" => "as you would a library"

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Assume you have an existing iOS app at `some/path/MyApp`, and that you
+want your Flutter project as a sibling:
+
+```bash
+cd some/path/
+flutter create -t module my_flutter
+```
+
+This creates a `some/path/my_flutter/` Flutter module project with some Dart
+code to get you started and a `.ios/` hidden subfolder. The `.ios/` folder wraps
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: "started and" => "started, and"

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+up the module project that contains some Cocoapods and a helper Ruby script.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: Delete "up".

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
+you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+of plugins read by the `podhelper.rb` script. Then run `pod install` again from
+`some/path/MyApp`.
+
+The `podhelper.rb` script ensures that your plugins, the Flutter.framework, and
+the App.framework are embedded in your project.
+
+You should now be able to build the project using `⌘B`.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: This advice is very mac-centric. Can we add the equivalent in Windows? (Here and elsewhere.)

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+### Under the hood
+
+If you have some reason to do this manually or debug why these steps aren't
+working, here's what's going on under the hood:
+
+- `Flutter.framework` (the Engine library) is getting embedded into your app for you.  This has to match up with the release type
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/project-setup.md]: From here on, your lines are kind of long, but I can fix that later.

**Comment by xster** [comment]
Thanks Shams

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 1 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: done

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 2 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: done. (it seems like it was only spelled this way in this file)

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 3 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: changed to "as a library"

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 4 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: thanks

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 5 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: done

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 6 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: Adding an iOS module to an iOS project is only possible on a mac.

**Pull request merged**